src/features/assets/hooks/useAssetPolling.ts

```
@@ -0,0 +1,47 @@
+      const response = await getAssetDetail(assetId);
+      const data = response.result;
+      setAsset(data);
+
+      // OCR 상태가 'completed'거나 'failed'면 폴링 중단
+      if (data.ocrStatus === "completed" || data.ocrStatus === "failed") {
+        setIsLoading(false);
+        return;
+      }
+
+      // 아직 처리 중이면 설정된 간격(interval) 후에 다시 호출
+      setTimeout(fetchStatus, interval);
```

ESLint

[error] Error: Cannot access variable before it is declared

`fetchStatus` is accessed before it is declared, which prevents the earlier access from updating when this value changes over time.

/home/jailuser/git/src/features/assets/hooks/useAssetPolling.ts:25:18
  23 |
  24 |       // 아직 처리 중이면 설정된 간격(interval) 후에 다시 호출
> 25 |       setTimeout(fetchStatus, interval);
     |                  ^^^^^^^^^^^ `fetchStatus` accessed before it is declared
  26 |     } catch (err) {
  27 |       setError("자산 정보를 불러오는 중 오류가 발생했다.");
  28 |       setIsLoading(false);

/home/jailuser/git/src/features/assets/hooks/useAssetPolling.ts:10:3
   8 |   const [error, setError] = useState<string | null>(null);
   9 |
> 10 |   const fetchStatus = useCallback(async () => {
     |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 11 |     if (!assetId) return;
     | ^^^^^^^^^^^^^^^^^^^^^^^^^
> 12 |
     …
     | ^^^^^^^^^^^^^^^^^^^^^^^^^
> 29 |     }
     | ^^^^^^^^^^^^^^^^^^^^^^^^^
> 30 |   }, [assetId, interval]);
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^ `fetchStatus` is declared here
  31 |
  32 |   useEffect(() => {
  33 |     if (assetId) {

(react-hooks/immutability)